peer_mentoring/account_management/views.py

```diff
@@ -21,6 +11,12 @@ def home(request):
     return render(request, "account_management/home.html")
 
 
+def view_profile(request, user_id):
+    user = UserProfile.objects.get(pk=user_id)
```

Comment:
probably safer to use `get_object_or_404` (django.shortcuts) in case the user does not exist, because then this code would raise an UserProfile.DoesNotExist exception